web/downloader.py

```diff
@@ -0,0 +1,23 @@
+from shutil import move
+from os import listdir
+from requests import get
+from pathlib import Path
+from vosk import Model
+
+
+class Downloader():
+
+    def download_vosk_model(model_name) -> None:
+        Model(model_name=model_name)
+        try:
+            move(Path.home() / f".cache/vosk/{model_name}", Path(".") / ".models")
+        except Exception:
+            print(f"{model_name} already exists")
```

Comment:
You don't need to download anything if the file already exists. Just check it at the start of the function.